static/locales/en-US.yaml

```diff
@@ -187,24 +187,24 @@ Settings:
       720p: 720p
       1080p: 1080p
       1440p: 1440p
-      4k: 4k
-      8k: 8k
+      4k: 4K
+      8k: 8K
   Privacy Settings:
     Privacy Settings: Privacy Settings
     Remember History: Remember History
     Save Watched Progress: Save Watched Progress
     Clear Search Cache: Clear Search Cache
     Are you sure you want to clear out your search cache?: Are you sure you want to
-      clear out your search cache?
+  clear out your search cache?
     Search cache has been cleared: Search cache has been cleared
     Remove Watch History: Remove Watch History
     Are you sure you want to remove your entire watch history?: Are you sure you want
-      to remove your entire watch history?
+  to remove your entire watch history?
     Watch history has been cleared: Watch history has been cleared
     Remove All Subscriptions / Profiles: Remove All Subscriptions / Profiles
     Are you sure you want to remove all subscriptions and profiles?  This cannot be undone.: Are
-      you sure you want to remove all subscriptions and profiles?  This cannot be
-      undone.
+  you sure you want to remove all subscriptions and profiles? This cannot be
+  undone.
   Subscription Settings:
     Subscription Settings: Subscription Settings
     Hide Videos on Watch: Hide Videos on Watch
```

Comment:
```suggestion
    Hide Videos on Watch: Hide Watched Videos
```
?

Reply:
The thing here is that it only hides watched videos on the watch page. So that was probably the intention behind 'Watch' as in Watch page back when profiles were not back in again. But I would say that this now fits better.  Once could add a tooltip which shortly states 'Only hides watched videos on subscription page' in order to make it clear.

Reply:
So, what should I opt to put in?

Reply:
Yes, you can leave it like it is. I'll add the tooltip afterwards :)

Reply:
Changed it around